configure.php

```diff
@@ -0,0 +1,528 @@
+			'Create WordPress Theme'       => str_replace( '_', ' ', title_case( $theme_slug ) ),
+			'CREATE_WORDPRESS_THEME'       => strtoupper( str_replace( '-', '_', $theme_slug ) ),
+			'create_wordpress_theme'       => str_replace( '-', '_', $theme_slug ),
+			'Create_WordPress_Theme'       => $theme_namespace,
+		],
+	);
+}
+
+if ( ! empty( $plugin_slug ) ) {
+	$search_and_replace = array_merge(
+		$search_and_replace,
+		[
+			'create-wordpress-plugin'      => $plugin_slug,
+			'Create WordPress Plugin'      => str_replace( '_', ' ', title_case( $plugin_slug ) ),
+			'CREATE_WORDPRESS_PLUGIN'      => strtoupper( str_replace( '-', '_', $plugin_slug ) ),
+			'create_wordpress_plugin'      => str_replace( '-', '_', $plugin_slug ),
+			'Create_WordPress_Plugin'      => $plugin_namespace,
+		]
+	);
+}
+
+run(
+	'composer config extra.wordpress-autoloader.autoload --json \'' . json_encode( [
+		$plugin_namespace => "plugins/{$plugin_slug}/src",
+		// $theme_namespace  => "themes/{$theme_slug}/src",
+	] ) . '\'',
+);
+
+if ( ! empty( $plugin_slug ) ) {
+	write( "Scaffolding create-wordpress-plugin to plugins/{$plugin_slug}..." );
+
+	run(
+		"composer create-project alleyinteractive/create-wordpress-plugin plugins/{$plugin_slug}",
+		$current_dir,
+	);
+
+	run( "mv plugins/{$plugin_slug}/plugin.php plugins/{$plugin_slug}/{$plugin_slug}.php" );
+
+	echo "Done!\n\n";
+}
+
+// TODO: scaffold the theme from create-wordpress-theme when the project is ready.
+// if ( ! empty( $theme_slug ) ) {
+// 	write( "Scaffolding create-wordpress-theme to themes/{$theme_slug}..." );
+
+// 	run(
+// 		"composer create-project alleyinteractive/create-wordpress-theme themes/{$theme_slug}",
+// 		$current_dir,
+// 	);
+// }
+
+foreach ( list_all_files_for_replacement() as $path ) {
+	echo "Updating $path...\n";
+
+	replace_in_file( $path, $search_and_replace );
+
+	if ( str_contains( $path, 'README.md' ) ) {
+		remove_readme_paragraphs( $path );
+	}
+}
+
+echo "Done!\n\n";
+
+write( 'Running composer update...' );
+
+run( 'composer update' );
+
+// Move the ci-templates to the root of the project.
+if ( is_dir( 'ci-templates' ) ) {
+	write( "Moving ci-templates files to the root of the project and removing ci-templates directory..." );
+
+	run( 'rsync -a ci-templates/ ./ && rm -rf ci-templates' );
+}
+
+// Remove admins from the CODEOWNERS file (should only apply to the template).
+write( 'Removing alleyinteractive/admins from .github/CODEOWNERS...' );
+replace_in_file(
+	'.github/CODEOWNERS',
+	[
+		'*	@alleyinteractive/admins' => '',
+	]
+);
+
+write( 'Removing configuration script from theme/plugin...' );
+
+delete_files(
+	[
+		"themes/{$theme_slug}/configure.php",
+		"themes/{$theme_slug}/Makefile",
+		"plugins/{$plugin_slug}/configure.php",
+		"plugins/{$plugin_slug}/Makefile",
+	]
+);
+
+echo "Done!\n\n";
+
+if ( confirm( 'Will this project be using Buddy CI?', true ) ) {
+	echo "Deleting GitHub Actions workflows...\n";
+
+	delete_files( '.github/workflows' );
```

Comment:
To allow flexibility for other CI systems in the future (e.g., CircleCI, something else) this should be a "delete all CI files except" pattern, which likely needs to be abstracted to a function that could take a parameter of what type of files to keep, if any, and if blank, would delete all CI files.